### PR TITLE
mapserver: fix RPATHs

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

See #132875.

### CODE REVIEW

**Comment by carlocab** [comment]
Oops, doesn't work. I'll have to look at their CMake scripts again...

File: Formula/mapserver.rb [hunk 1]
```diff
@@ -44,12 +44,25 @@ def python3
...
 
+  # Respect configuration of `CMAKE_INSTALL_RPATH`
+  # https://github.com/MapServer/MapServer/pull/6902
+  patch do
+    url "https://github.com/MapServer/MapServer/commit/6fbfecc81d4eaca193edad97c4c0b64baf308554.patch?full_index=1"
+    sha256 "be4e0c35a29210b08d99adb662c6a72a39e969ec5dfdb122bfda1736df777f0b"
+  end
+
+  # Fix mapscript Python bindings RPATH
+  patch :DATA
```
Comment (carlocab) [inline Formula/mapserver.rb:55]: Will upstream if this works.

**Comment by chenrui333** [comment]
@carlocab is this good to go?

**Comment by chenrui333** [comment]
gonna close it for now

**Pull request closed without merging**